packages/picasso-rich-text-editor/src/RichText/RichText.tsx

```diff
@@ -24,7 +25,7 @@ export const RichText = ({
     <Container
       style={style}
       data-testid={dataTestId}
-      className={className}
+      className={twJoin('break-all', className)}
```

Comment:
```suggestion
      className={twJoin('break-words', className)}
```
This should [preserve](https://tailwindcss.com/docs/word-break) words that would still fit, but break longer words that would overflow.